modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/episode/EpisodeFragment.kt

```diff
@@ -170,6 +207,9 @@ class EpisodeFragment : BaseDialogFragment() {
 
     override fun onDestroyView() {
         super.onDestroyView()
+        if (!viewModel.isFragmentChangingConfigurations) {
+            analyticsTracker.track(AnalyticsEvent.EPISODE_DETAIL_DISMISSED)
```

Comment:
I think this is supposed to have a "source" property.

Reply:
Added in 237b285